src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,375 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import {
+    CloudServiceProvider,
+    CreateRequest,
+    ServiceDetailVo,
+    ServicesAvailableService,
+    ServiceService,
+    ServiceVo,
+    UserAvailableServiceVo,
+} from '../../../../xpanse-api/generated';
+import { SelectDestination } from './SelectDestination';
+import { ShowDeploy } from './ShowDeploy';
+import { Button, Popconfirm, Space, Steps } from 'antd';
+import { DestroyService } from './DestroyService';
+import { OrderSubmitProps } from '../OrderSubmit';
+import { deployTimeout, destroyTimeout, usernameKey, waitServicePeriod } from '../../../utils/constants';
+import OrderSubmitResult from '../OrderSubmitResult';
+import OrderSubmitFailed from '../OrderSubmitFailed';
+import { getCreateRequest, getDeployParams, OperationType } from '../formElements/CommonTypes';
+import { ProcessingStatus } from '../ProcessingStatus';
+import { ExportServiceData } from './ExportServiceData';
+import { ImportServiceData } from './ImportServiceData';
+
+export const MigratingServices = ({
+    record,
+    getCloseStatus,
+}: {
+    record: ServiceVo | undefined;
+    getCloseStatus: (isClose: boolean) => void;
+}): JSX.Element => {
+    const formRef = useRef<HTMLFormElement | null>(null);
+    const [current, setCurrent] = useState(0);
+    const [userAvailableServiceVoList, setUserAvailableServiceVoList] = useState<UserAvailableServiceVo[]>([]);
+
+    const [selectCsp, setSelectCsp] = useState<CloudServiceProvider.name | undefined>(undefined);
+    const [selectArea, setSelectArea] = useState<string>('');
+    const [selectRegion, setSelectRegion] = useState<string>('');
+    const [selectFlavor, setSelectFlavor] = useState<string>('');
+
+    const [tip, setTip] = useState<JSX.Element | undefined>(undefined);
+    const [deploying, setDeploying] = useState<boolean>(false);
```

Comment:
add `isXXXXXX` to all boolean variables pls.

Reply:
fixed